component/signature.mjs

```diff
@@ -84,7 +84,7 @@ const parseBabelLog = (content, url) => {
     return parseBabel(content, {
       sourceType: "module",
       sourceFilename: url,
-      plugins: ["estree"],
+      plugins: [["estree", { classFeatures: true }]],
```

Comment:
FYI This file is used to build the agent and it does not contain any `PropertyDefinition`. But it is fine to add it as well here.

Reply:
I figured, but it turned out in grep and I figured it won't hurt for completeness ;)